site/en/install/gpu.md

```diff
@@ -98,15 +98,15 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
 # Install development and runtime libraries (~4GB)
 <code class="devsite-terminal">sudo apt-get install --no-install-recommends \
     cuda-10-0 \
-    libcudnn7=7.4.1.5-1+cuda10.0  \
-    libcudnn7-dev=7.4.1.5-1+cuda10.0
+    libcudnn7=7.6.0.64-1+cuda10.0  \
+    libcudnn7-dev=7.6.0.64-1+cuda10.0
 </code>
 
 # Install TensorRT. Requires that libcudnn7 is installed above.
 <code class="devsite-terminal">sudo apt-get update && \
         sudo apt-get install nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
         && sudo apt-get update \
-        && sudo apt-get install -y --no-install-recommends libnvinfer-dev=5.0.2-1+cuda10.0
+        && sudo apt-get install -y --no-install-recommends libnvinfer-dev=5.1.5-1+cuda10.0
```

Comment:
Have you tried this? I believed we don't need to install `nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0` above for 5.1.5.

Reply:
That may also have changed.  We need to give it a try on a fresh instance or fresh docker.  It was was annoying that you had to install that repo so I would not be shocked it is fixed.  I will add a comment ad that will be the last thing to resolve.